src/consumer/component/link-generator.js

```diff
@@ -23,13 +23,27 @@ const LINKS_CONTENT_TEMPLATES = {
   less: "@import '{filePath}.less';"
 };
 
+const PACKAGES_LINKS_CONTENT_TEMPLATES = {
+  css: "@import '~{filePath}.css';",
+  scss: "@import '~{filePath}.scss';",
+  sass: "@import '~{filePath}.sass';",
+  less: "@import '~{filePath}.less';"
+};
+
+const fileExtentions = ['js', 'ts', 'jsx', 'tsx'];
```

Comment:
change this to something more meaningful, these extensions are relevant only for specific process (npm) not a general fileExtensions